hardhat.config.ts

```diff
@@ -15,6 +16,182 @@ const accounts: HDAccountsUserConfig = {
     'test test test test test test test test test test test junk',
 }
 
+const SUPPORTED_NETWORKS = [
+  'celo',
+  'mainnet',
+  'arbitrum',
+  'polygon',
+  'op',
+  'base',
+  'berachain',
+  'vana',
+]
+
+const ONE_DAY = 60 * 60 * 24
+
+task('deploy-registry', 'Deploy Registry contract')
```

Comment:
nit: can you move these to a file in a `tasks/` subdir? We do that w/ other projects, like valora-punks (dubious reference, I know).